src/TemperatureConverter.spec.ts

```diff
@@ -0,0 +1,67 @@
+import { Scales, TemperatureConverter } from './TemperatureConverter';
+
+describe('Convert temperature', () => {
```

Comment:
Pode tirar os async dos métodos it, devido a função não ser assíncrona.

Reply:
Verdade, vou tirar!